code/__HELPERS/priority_announce.dm

```diff
@@ -44,6 +44,39 @@
 			if(M.client.prefs.toggles & SOUND_ANNOUNCEMENTS)
 				SEND_SOUND(M, s)
 
+/**
+ * Summon the crew for an emergency meeting
+ *
+ * Teleports the crew to a specified area, and tells everyone (via an announcement) who called the meeting. Should only be used during april fools!
+ * Arguments:
+ * * user - Mob who called the meeting
+ * * button_zone - Area where the meeting was called and where everyone will get teleported to
+ */
+/proc/call_emergency_meeting(mob/living/user, area/button_zone)
+	var/announcement
+
+	announcement += "<h1 class='alert'>Captain Alert</h1>"
+	//If the announcer overrides alert messages, use that message.
+	announcement += "<br><span class='alert'>[user] has called an Emergency Meeting!</span><br>"
+	announcement += "<br>"
+
+	var/s = sound('sound/misc/emergency_meeting.ogg')
```

Comment:
im working off the assumption you're expecting this to get merged

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names